exec/java-exec/src/main/codegen/data/DecimalAggrTypes1.tdd

```diff
@@ -35,6 +35,11 @@
        {inputType: "VarDecimal", outputType: "NullableVarDecimal"},
        {inputType: "NullableVarDecimal", outputType: "NullableVarDecimal"}
       ]
+   },
+   {className: "AnyValue", funcName: "any_value", types: [
+       {inputType: "VarDecimal", outputType: "NullableVarDecimal"},
+       {inputType: "NullableVarDecimal", outputType: "NullableVarDecimal"}
```

Comment:
Could you please also add agg function which takes repeated VarDecimal, since it was done for other types?

Reply:
I am unable to test it since casting repeated varchars is not yet supported. So, I have disabled the test for now.

Reply:
Modified testcase to use the parquet file.